tests/common/utilities.py

```diff
@@ -488,7 +488,10 @@ def compare_crm_facts(left, right):
     for k, v in list(left['resources'].items()):
         lv = v
         rv = right['resources'][k]
-        if lv['available'] != rv['available'] or lv['used'] != rv['used']:
+
+        # For Cisco-8000 devices, hardware counters are statistical-based with +/- 1 entry tolerance.
+        # Hence, the available counter may not increase as per initial value.
+        if abs(int(lv['available']) - int(rv['available'])) > 1 or abs(int(lv['used']) - int(rv['used'])) > 1:
```

Comment:
Can we add a platform check here? Like if the platform is Cisco-8000, we check the counter change with +/- entry tolerance?